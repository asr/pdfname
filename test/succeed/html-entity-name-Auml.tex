\documentclass{article}

% The real example is
% https://www.cambridge.org/core/journals/journal-of-functional-programming/article/why-the-constant-undefined-logics-of-partial-terms-for-strict-and-nonstrict-functional-programming-languages/3351BE72FC143DEEC339C155D7910A0B.

\pdfinfo{%
  /Author (ST\&Auml;RK)%
  /Title  (ST\&Auml;RK)%
  /CreationDate (D:2017)%
}

\begin{document}
pdfname test.
\end{document}
