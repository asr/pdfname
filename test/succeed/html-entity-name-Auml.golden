ret > ExitSuccess
out > The full path name will be /tmp/stark-2017.stark.pdf
out >
